exporter/zipkin/zipkin.go

```diff
@@ -29,6 +33,11 @@ import (
 type Exporter struct {
 	reporter      reporter.Reporter
 	localEndpoint *model.Endpoint
+
+	// Logger will be used to report errors from this exporter.
+	Logger interface {
```

Comment:
Can we instead have hook like onError at https://godoc.org/contrib.go.opencensus.io/exporter/stackdriver#Options?

It is useful for those who wants to report the errors to a service.